.tidyallrc

```diff
@@ -1,3 +1,6 @@
 [PerlTidy]
-select = **/*.{pl,pm,t}
+; Notice that this does only include Perl modules and tests, not scripts in
+; the scripts/ directory where we also have non-Perl files. Use tools/tidy to
+; cover all files
+select = **/*.{pl,pm,t} script/*
```

Comment:
The comment says "not scripts in the scripts/ directory" but the code looks pretty much like it would include those scripts.

Reply:
ah, right. The comment was outdated. I fixed that. I only updated the comment, no change in functionality

Reply:
So despite `script/*` staying in the code it is still "suggested to use tools/tidy to cover all appropriate files"?

Reply:
Yes, unfortunately we need to still mention `script/*` here. I explained that in https://github.com/os-autoinst/openQA/pull/4485/commits/64be93648acd98f190a3c489fdd6b065bc01962c : We still need to include the script directory in
.tidyallrc so that it's not excluded even if tidyall is called with an explicit file list as arguments.